core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -1256,37 +1255,37 @@ public void testTableAuth(
             required(2, "data", Types.StringType.get()));
 
     if (requiresNamespaceCreate()) {
-      catalog.createNamespace(ident.namespace());
+      catalog.createNamespace(TBL.namespace());
     }
 
-    Table table = catalog.createTable(ident, expectedSchema);
+    Table table = catalog.createTable(TBL, expectedSchema);
     assertThat(table.schema().asStruct())
         .as("Schema should match")
         .isEqualTo(expectedSchema.asStruct());
 
-    Table loaded = catalog.loadTable(ident); // the first load will send the token
+    Table loaded = catalog.loadTable(TBL); // the first load will send the token
     assertThat(loaded.schema().asStruct())
         .as("Schema should match")
         .isEqualTo(expectedSchema.asStruct());
 
     loaded.refresh(); // refresh to force reload
 
-    Mockito.verify(adapter)
+    verify(adapter)
```

Comment:
I was about to merge but just saw this and this seems like an unrelated change that we typically wouldn't want to include as part of this PR. Could you please revert the places that removed `Mockito`?

Reply:
Looks like Intellij Idea did a refactoring without telling me :D I reverted those changes and apply only ident changes.